agate-webapp/src/main/webapp/app/services.js

```diff
@@ -188,47 +188,22 @@ agate.factory('AuthenticationSharedService', ['$rootScope', '$http', '$log', '$c
         });
       },
       isAuthenticated: function () {
-        // WORKAROUND: until next angular update, cookieStore always returns NULL event 'agatesid' cookie exists
-        function getAgateSidCookie() {
-          var regexp = /agatesid=([^;]+)/g;
-          var result = regexp.exec(document.cookie);
-          return (result === null) ? null : result[1];
-        }
-
-        if (!getAgateSidCookie()) {
-          // session has terminated, cleanup
-          Session.destroy();
-          return false;
-        }
+        if (!Session.login && $cookies.get('agate_subject')) {
+          var account;
 
-        if (!Session.login) {
-          // check if the user has a cookie
-          if ($cookies.get('agate_subject')) {
-            var account;
-
-            try {
-              account = JSON.parse($cookies.get('agate_subject'));
-            } catch (e) {
-              $log.info('Invalid agate_subject cookie value. Ignoring.');
-            }
-
-            if (account) {
-              Session.create(account.login, account.role, account.realm);
-              $rootScope.account = Session;
-            }
-          } else if ($cookies.get('agatesid')) {
-            // Open ID case
```

Comment:
Does it breaks the OpenID case?

Reply:
the cookie is still present in the request. this means that the user will be authenticated by the server, client will no longer be able to interact with it.